parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -1,3 +1,16 @@
 #!/bin/bash
 
-# Placeholder for future CIS-related shell utilities
\ No newline at end of file
+assignRootPW() {
+  grep '^root:[!*]:' /etc/shadow
+  if [ $? -eq '0' ] ; then
```

Comment:
This could be simplified to:
`if grep '^root:[!*]:' /etc/shadow; then`

Reply:
punt on this change